specification/cognitiveservices/data-plane/AnomalyDetector/preview/v2.0-preview/RootCauseAnalysis.json

```diff
@@ -0,0 +1,2143 @@
+      "required": [
+        "rootCauses"
+      ],
+      "type": "object",
+      "properties": {
+        "rootCauses": {
+          "description": "root causes",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MultiDimRootCause"
+          }
+        }
+      }
+    },
+    "RootCauseAnalysisInferenceResponse": {
+      "required": [
+        "createdDateTime",
+        "inferenceId",
+        "instanceId",
+        "kind",
+        "lastModifiedDateTime",
+        "modelId",
+        "status"
+      ],
+      "type": "object",
+      "properties": {
+        "kind": {
+          "description": "kind of root cause analysis",
+          "enum": [
+            "multidimensional"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "RootCauseAnalysisType",
+            "modelAsString": true
+          }
+        },
+        "inferenceId": {
+          "format": "uuid",
+          "description": "Inference unique id for root cause analysis",
+          "type": "string"
+        },
+        "instanceId": {
+          "format": "uuid",
+          "description": "Instance unique id for root cause analysis",
+          "type": "string"
+        },
+        "modelId": {
+          "format": "uuid",
+          "description": "Model unique id for root cause analysis",
+          "type": "string"
+        },
+        "status": {
+          "description": "Inference task status for root cause analysis",
+          "enum": [
+            "scheduled",
+            "running",
+            "succeeded",
+            "failed",
+            "canceled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "RootCauseAnalysisInferenceStatus",
+            "modelAsString": true
+          }
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "Inference task created date time",
+          "type": "string"
+        },
+        "lastModifiedDateTime": {
+          "format": "date-time",
+          "description": "Inference task state modified date time",
+          "type": "string"
+        },
+        "error": {
+          "$ref": "#/definitions/MultiDimRCACreateDataSetErrorDetail"
+        },
+        "result": {
+          "$ref": "#/definitions/MultiDimRCAInferenceResult"
+        }
+      },
+      "discriminator": "kind"
+    },
+    "MultiDimRCAInferenceResponse": {
+      "required": [
+        "parameter"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RootCauseAnalysisInferenceResponse"
+        }
+      ],
+      "properties": {
+        "parameter": {
+          "$ref": "#/definitions/MultiDimRCAInferenceParameter"
+        }
+      },
+      "x-ms-discriminator-value": "multidimensional"
+    }
+  },
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
```

Comment:
Please include a description in the securityDefinition that can be shown in the generated REST API reference docs.
```
 "description": "The secret key for your Azure Cognitive Services subscription."
```